app/components/meetings/api.js

```diff
@@ -272,6 +272,45 @@ export default class Api
     await axios.put(url, file, config);
   }
 
+  async getMeetingDocuments(options) {
+    const documents = await this.http.get(`/api/v2016/documents`, options).then(res => res.data).catch(tryCastToApiError);
+    return documents;
+  } 
+
+  async getMeetings(options) {
+    const meetings = await this.http.get(`/api/v2016/meetings`, options).then(res => res.data).catch(tryCastToApiError);
+    return meetings;
+  }
```

Comment:
same ad for queryMeetingDocuments   

```suggestion
  async queryMeetings({cache, ...params} ) {
  
  cache = !!cache; // force boolean. default false
  
    const meetings = await this.http.get(`/api/v2016/meetings`, { params, cache }).then(res => res.data).catch(tryCastToApiError);
    return meetings;
  }
```

Reply:
Since getMeetings is being used somewhere else, I created a new function called queryMeetings using the same concept as applied on the queryMeetingDocuments.